### DISCUSSION TITLE
Divine price not listed top left of pricecheck screen.

### DISCUSSION THREAD

**Discussion by Manshima** [SnosMe/awakened-poe-trade#966] [Q&A]
I can't seem to find the setting for this and was hoping somebody could point me in the right direction. Divine price up the top left is no longer showing with the latest version. Have reinstalled and checked settings - i'm unable to find it.
Would greatly appreciate any help. 
Thank you

**Comment by syla3** [comment]
same iv reinstalled a few times 

![image](https://user-images.githubusercontent.com/131329792/233255850-390bf8c9-f39b-446d-b2a6-31042dfb0492.png)

**Comment by Ruebane** [comment]
same hope they answer you with a fix ill be watching.

**Reply by SnosMe** [reply]
don't wait, because I can't fix ur network (the reason why it fails)

**Reply by Ruebane** [reply]
Do you know what kinda of network issue it is so i can try to look into it and fix it on my end because ive turn all my firewalls and virus protection off and my internet is running at over 100mps?

**Reply by SnosMe** [reply]
it's super weird, i've not encountered it myself, and hard to tell from reports I've seen so far.

let's see what you have, open APT in browser 
![image](https://github.com/SnosMe/awakened-poe-trade/assets/4292308/1b4ba859-8f30-4888-a83d-f3a7f4139a2d)

then open console (F12), refresh the page, take the screenshot of error

**Reply by Ruebane** [reply]
![image](https://github.com/SnosMe/awakened-poe-trade/assets/156227608/a5a405fa-35ba-4519-93aa-917f31a05708)

Hope the image works but this is what it says

**Comment by Manshima** [comment]
Spotted this old post being revived. Same issue is persisting intermittently and i've still found no resolution. It worked at the start of 3.23 for about a week, then it died.
I'll post my error in here to try and help get to the bottom of it. Happy to help with any other information needed
![APT error](https://github.com/SnosMe/awakened-poe-trade/assets/131322112/6e899491-8c63-43c5-b87f-dddaaa96d357)
